pkg/controllers/dynakube/extension/conditions.go

```diff
@@ -0,0 +1,39 @@
+package extension
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	secretConditionType       = "ExtensionsTokenSecret"
+	secretCreatedReason       = "SecretCreated"
+	secretCreatedMessageTrue  = "EEC token created"
+	secretCreatedMessageFalse = "Error creating extensions secret: %s"
+)
+
+func setSecretCreatedTrue(conditions *[]metav1.Condition) {
+	condition := metav1.Condition{
+		Type:    secretConditionType,
+		Status:  metav1.ConditionTrue,
+		Reason:  secretCreatedReason,
+		Message: secretCreatedMessageTrue,
+	}
+	_ = meta.SetStatusCondition(conditions, condition)
+}
+
+func setSecretCreatedFalse(conditions *[]metav1.Condition, err error) {
+	condition := metav1.Condition{
+		Type:    secretConditionType,
+		Status:  metav1.ConditionFalse,
+		Reason:  secretCreatedReason,
+		Message: fmt.Sprintf(secretCreatedMessageFalse, err),
+	}
+	_ = meta.SetStatusCondition(conditions, condition)
+}
+
+func removeSecretCreated(conditions *[]metav1.Condition) bool {
```

Comment:
could you please change the name? The `extension.removeSecrtedCreated([]conditions)` name suggests that it removes a K8S secret based on conditions..